File: algorithm/leetcode660.cpp
class Solution {
public:
    int newInteger(int n) {
    	vector<int> nums;
    	while(n)
    	{
    		nums.push_back(n%9);
    		n /= 9;
    	}
    	int ans = 0;
    	for(int i = nums.size()-1; i >= 0; --i)
    		ans = ans * 10 + nums[i];
    	return ans;
    }
};

File: algorithm/Python3_version/leetcode660.py
class Solution:
    def newInteger(self, n: int) -> int:
        num = []
        while n:
            num.append(n%9)
            n //= 9
        ans = 0
        for i in range(len(num)-1,-1,-1):
            ans = ans*10+num[i]
        return ans